components/eppp_link/examples/emac2emac/README.md

```diff
@@ -0,0 +1,12 @@
+
+# EPPP link with EMAC to EMAC transport layer
+
+This example runs a symmetrical server-client eppp application with iperf component using Ethernet transport layer, with a customized Ethernet driver from [eth_dummy_phy](https://components.espressif.com/components/espressif/eth_dummy_phy) component.
+
+Please refer to the component documentation for more information about the principle of operation and the actual physical connection between nodes.
+
+## How to use this example
+
+* Choose `CONFIG_EXAMPLE_NODE_SERVER` for the device connected as **RMII CLK Source Device**
```

Comment:
We should add some advice here regarding clk source. 
@kostaond the conditions for clk generation if Wi-Fi is used in the device applies here as well, right?